jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Query.java

```diff
@@ -116,11 +126,19 @@ public boolean isAllowDottedKeys() {
 		return allowDottedKeys;
 	}
 
+	public boolean isUseAllTypeNames() {
```

Comment:
Sorry, got distracted in the middle of the review. I agree, consistency is key. Those accessors are part of the external contract (they define the configuration format), so changing them is probably not a good idea. So forget my comment and keep the current naming. Or find a naming that is coherent with both English and Java Bean convention and consistent with current configuration format.

:+1: